src/app/event/[eventId]/(eventId)/EventTimeline.tsx

```diff
@@ -0,0 +1,23 @@
+import Calendar from '@/components/elements/Timeline/Calendar'
+import { Box } from '@mui/material'
+
+export const EventTimeline: React.FC<{
+  id: string
+  budget: number
+  date: string
+  objective: string
+  attendees: number
+  theme: string
+  services: string
+}> = ({ id, budget, date, objective, attendees, theme, services }) => {
+  return (
+    <Box
+      alignItems={'center'}
+      justifyContent={'center'}
+      padding={{ xs: '2em 2em', lg: '2em 12em' }}
+    >
+      <hr className="mb-8" />
+      <Calendar eventId={id} />
+    </Box>
```

Comment:
I think, you don't have to define everything here (you don't have to define budget, date, etc) because it's not needed. This may cause unused variables that will be technical debts in the future.

Reply:
Thanks for the input , unused variable removed!